configure.py

```diff
@@ -2313,11 +2313,21 @@ def test_exe_extra_ldflags():
         variables['includedir'],
         'botan-%d' % (Version.major()), 'botan')
 
-    if cc.basename == 'msvc' and variables['cxx_abi_flags'] != '':
-        # MSVC linker doesn't support/need the ABI options,
-        # just transfer them over to just the compiler invocations
+    # On MSVC, the "ABI flags" should be passed to the compiler only, on other platforms, the
```

Comment:
These shenanigans seem to suggest that we might end up with something that builds in CI but not on a developers machine, which isn't helpful. Is the issue here that Clang really dislikes `/MD`/etc during the link step?

Reply:
This is relevant to https://github.com/randombit/botan/issues/4424.